app/controllers/articles_controller.rb

```diff
@@ -327,6 +328,12 @@ def article_params_json
       allowed_params << :organization_id
     end
 
+    manage_published_at_params
```

Comment:
Not a great improvement, I just moved the code to a separate method to please rubocop. I would prefer a more serious refactoring, but that's a separate task.